fileinfo_test.go

```diff
@@ -131,3 +132,49 @@ func TestGetFileStandardInfo_Directory(t *testing.T) {
 	}
 	checkFileStandardInfo(t, info, expectedFileInfo)
 }
+
+// TestFileInfoStructAlignment checks that the alignment of Go fileinfo structs
+// match what is expected by the Windows API.
+func TestFileInfoStructAlignment(t *testing.T) {
+	//nolint:revive // SNAKE_CASE is not idiomatic in Go, but aligned with Win32 API.
+	const (
+		exampleLARGE_INTEGER uint64 = 0
+		exampleULONGLONG     uint64 = 0
+	)
```

Comment:
since `LARGE_INTEGER` is a union with `LONGLONG`, can we replace these two with:
```go
const align64 = unsafe.Alignof(uint64(0))
```

Reply:
The idea here was to match the type name in the Windows docs with a Go type that represents it to make it easier to compare/review. Less time spent translating. Writing out the `unsafe.Alignof` here rather than later doesn't interfere with that idea though, I'll try that.